<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>876</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="2">
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>640</width>
        <height>480</height>
       </size>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="gbFilters">
      <property name="title">
       <string>Filters</string>
      </property>
      <widget class="QRadioButton" name="rbNoneFlt">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>102</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>None</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="rbGrayFlt">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>102</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Grayscale</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Controls</string>
      </property>
      <widget class="QPushButton" name="btnOpen">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>50</y>
         <width>116</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Open Video</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnStop">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>80</y>
         <width>71</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnPlay">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>67</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Play</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnPause">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>110</y>
         <width>114</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Pause</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnConvert">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>33</x>
         <y>150</y>
         <width>121</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Convert To File</string>
       </property>
      </widget>
      <widget class="QProgressBar" name="pbConvert">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>210</y>
         <width>118</width>
         <height>23</height>
        </rect>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>876</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
